cpp/ps0001/ps0001_soin_cpp.cpp

```diff
@@ -1,15 +1,19 @@
-#include<iostream>
+#include <iostream>
 #include <conio.h>
 using namespace std;
 int main()
 {
-	int x, y, z, f;
+	// prompting users for 3 numbers to add 1/2
+	unsigned short x, y, z, f;
 	cin >> x;
 	cin >> y;
 	cin >> z;
+
+	// outputting the result of the summation process 3/3
 	f = x + y + z;
 	cout << "_______" << endl;
 	cout << f;
 	_getch();
 
-}
\ No newline at end of file
+}
```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [x] output is as required
```
155
940
168
--------
1263
```